comp/otelcol/otlp/components/exporter/serializerexporter/config.go

```diff
@@ -22,264 +18,22 @@ type ExporterConfig struct {
 	exporterhelper.QueueConfig   `mapstructure:",squash"`
 
 	Metrics MetricsConfig `mapstructure:"metrics"`
-
-	warnings []string
 }
 
 var _ component.Config = (*ExporterConfig)(nil)
 
 // MetricsConfig defines the metrics exporter specific configuration options
 type MetricsConfig struct {
-	// Enabled reports whether Metrics should be enabled.
-	Enabled bool `mapstructure:"enabled"`
-
-	// TCPAddr.Endpoint is the host of the Datadog intake server to send metrics to.
-	// If unset, the value is obtained from the Site.
-	confignet.TCPAddrConfig `mapstructure:",squash"`
-
-	// DeltaTTL defines the time that previous points of a cumulative monotonic
-	// metric are kept in memory to calculate deltas
-	DeltaTTL int64 `mapstructure:"delta_ttl"`
+	Metrics datadogconfig.MetricsConfig `mapstructure:",squash"`
 
-	ExporterConfig MetricsExporterConfig `mapstructure:",squash"`
+	// The following 3 configs are only used in OTLP ingestion and not expected to be used in the converged agent.
```

Comment:
To confirm there is no way to set them on OTel Agent Exporter rt

Reply:
Theoretically they can set those in their config like:
```
exporters:
  datadog:
    metrics:
      tag_cardinality: ...
      tags: ...
      apm_stats_receiver_addr: ...
```

But as we discussed they should not set them in the converged agent, so after this PR they will not be able to. Also these 3 configs are not documented anywhere in converged agent.

(Agent OTLP ingestion is not affected.)